test/functional/sqlsrv/TC43_FetchData.phpt

```diff
@@ -91,16 +105,27 @@ function checkData($col, $actual, $expected)
     return ($success);
 }
 
-if (! isWindows()) {
-    setUTF8Data(true);
+if ( !isWindows() ) {
+    setlocale(LC_ALL, "en_US.ISO-8859-1");
+}
+
+// test ansi
+try {
+    setUTF8Data(false);
+    fetchFields();
+} catch (Exception $e) {
+    echo $e->getMessage();
 }
+
+// test utf8
 try {
+    setUTF8Data(true);
     fetchFields();
 } catch (Exception $e) {
     echo $e->getMessage();
 }
-setUTF8Data(false);
```

Comment:
By default, utf8data is off right? If you don't set it back to false at the end of the test, when the next test runs, the global uft8data will be true.